pynest/nest/lib/hl_api_simulation.py

```diff
@@ -234,14 +234,20 @@ def SetStructuralPlasticityStatus(params):
 
 
 @check_stack
-def GetStructuralPlasticityStatus(params):
+def GetStructuralPlasticityStatus(keys = None):
     """Get the current structural plasticity parameters for the network
     simulation.
     """
 
-    sps(params)
+    sps({})
     sr('GetStructuralPlasticityStatus')
-    return spp()
+    d = spp()
+    if keys is None:
+        return d
+    elif is_literal(keys):
+        return d[keys]
```

Comment:
Here, `keys` can only be a single key, so why the plural? And wouldn't it make more sense to allow a list of keys, as in `GetStatus`?